guvnor-ala/guvnor-ala-build-maven/src/test/java/org/guvnor/ala/build/maven/executor/MavenDependencyConfigExecutorTest.java

```diff
@@ -116,14 +118,14 @@ public void testMavenDependencyAPI() throws Exception {
                          binary.getArtifactId());
             assertEquals(version,
                          binary.getVersion());
-            assertEquals(m2Folder + "/org/guvnor/ala/maven-ala-artifact-test/1/maven-ala-artifact-test-1.pom",
+            assertEquals(Paths.get(m2Folder.getPath(), artifactPath).toString(),
```

Comment:
I think it should be possible to compare `Path` directly, without the `toString()` - no?

Reply:
The two Paths point at the same file, but are not actually equal as Path objects.

Reply:
Ah, my bad. I misunderstood what Path does.